x-pack/plugins/uptime/public/state/index.ts

```diff
@@ -12,6 +12,13 @@ import { rootReducer } from './reducers';
 
 export type AppState = ReturnType<typeof rootReducer>;
 
+type ComposeType = typeof compose;
+declare global {
+  interface Window {
+    __REDUX_DEVTOOLS_EXTENSION_COMPOSE__: ComposeType;
+  }
+}
+
 const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
```

Comment:
i would prefer to use this here, this package is already part of kibana, so it would be easier to use.

```
import { composeWithDevTools } from 'redux-devtools-extension';
export const store = createStore(rootReducer, composeWithDevTools(applyMiddleware(sagaMW)));
```